### PR TITLE
feat(PubSub): Add CPS to GCS sample

### PR DESCRIPTION
**Pull request by ajupazhamayil**
This PR add export of `GOOGLE_PUBSUB_STORAGE_BUCKET` to secrets.sh under .kokoro

### CODE REVIEW

**Comment by vishwarajanand** [comment]
Can you add a comment on what exactly was changed in the `.kokoro/secrets.sh.enc` file?
Also, before merging pull the latest encoded secrets file and add your changes there so that it doesnt override any changes since this PR was created.

File: pubsub/api/src/create_cloud_storage_subscription.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+use Google\Cloud\PubSub\PubSubClient;
+
+/**
+ * Creates a Pub/Sub GCS subscription.
+ *
+ * @param string $projectId  The Google project ID.
+ * @param string $topicName  The Pub/Sub topic name.
+ * @param string $subscriptionName  The Pub/Sub subscription name.
+ * @param string $tableName  The Cloud Storage bucket name.
+ *        The bucket name must be without any prefix like "gs://".
```
Comment (vishwarajanand) [inline pubsub/api/src/create_cloud_storage_subscription.php:36]:
```suggestion
 * @param string $tableName  The Cloud Storage bucket name without any prefix like "gs://".
```

**Review by vishwarajanand — COMMENTED** [review]
Lgtm, left some nits.
You might need to bump the pubsub version in `composer.json`.

File: pubsub/api/src/create_cloud_storage_subscription.php (hunk 1 above)
Comment (ajupazhamayil) [reply pubsub/api/src/create_cloud_storage_subscription.php]: Done

**Comment by ajupazhamayil** [comment]
> Can you add a comment on what exactly was changed in the `.kokoro/secrets.sh.enc` file? I believe it's `GOOGLE_PUBSUB_STORAGE_BUCKET` Also, before merging pull the latest encoded secrets file and add your changes there so that it doesn't override any changes since this PR was created.

Added to the description.

**Comment by ajupazhamayil** [comment]
Will wait for the release

**Comment by ajupazhamayil** [comment]
This has been released on Aug 5th! Lets proceed with the PR then.

**Review by vishwarajanand — APPROVED** [review]
Lgtm

**Pull request merged**